src/auro-popover.js

```diff
@@ -48,7 +49,8 @@ class AuroPopover extends LitElement {
   static get properties() {
     return {
       placement:  { type: String },
-      for:        { type: String }
+      for:        { type: String },
+      disabled:   { type: Boolean }
```

Comment:
We should put these in alphabetical order as a standard practice for all components